AAI/AAIConnectProfile.md

```diff
@@ -62,7 +62,7 @@ for the claim as this is a reference to a GA4GH documented standard only.
 service that provides to users an identity, authenticates it; and provides
 assertions to a Broker using standard protocols, such as OpenID Connect, SAML or
 other federation protocols. Example: eduGAIN, Google Identity, Facebook, NIH
-ERACommons. IdPs MAY be assertion sources.
+ERACommons. IdPs MAY be [Visa Assertion Sources](#term-visa-assertion-source).
```

Comment:
Assertion sources not otherwise defined so assuming it is meant to be reference to visa assertion sources